Firmware/language_pt.h

```diff
@@ -0,0 +1,343 @@
+/**
+ * Portuguese
+ *
+ * LCD Menu Messages
+ * Please note these are limited to 17 characters!
+ *
+ */
+
+#define(length=20) WELCOME_MSG              CUSTOM_MENDEL_NAME " pronta."
+#define MSG_SD_INSERTED                     "Cartao inserido"
+#define MSG_SD_REMOVED                      "Cartao removido"
+#define MSG_MAIN                            "Principal"
```

Comment:
In https://github.com/MarlinFirmware/Marlin/blob/1.1.x/Marlin/language_pt.h it is "Menu principal" or is yours a common description?

Reply:
Main -> Principal
Main menu -> Menu Principal
But OK, we can use Menu Principal